tests/unit_tests/EvalTests.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Catch/catch.hpp"
+#include "slang.h"
+
+using namespace slang;
+
+namespace {
+
+TEST_CASE("Simple eval", "[eval") {
```

Comment:
This looks like an awesome thing to have, this should be very useful for testing stuff.